### PR TITLE
Prepare v16.0.0 release

### PR DESCRIPTION
**Pull request by nikku**
This ensures we build upon up-to-date bpmn.io dependencies.

### CODE REVIEW

File: packages/dmn-js/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,19 @@ All notable changes to [dmn-js](https://github.com/bpmn-io/dmn-js) are documente
 
 ___Note:__ Yet to be released changes appear here._
 
+## 16.0.0
+
+* `CHORE`: remove broken touch interaction from DRD editor ([bpmn-io/diagram-js#796](https://github.com/bpmn-io/diagram-js/issues/796))
+* `DEPS`: update to `diagram-js@14.0.0`
```
[the remark this answers is not in the document]
Comment (nikku) [reply packages/dmn-js/CHANGELOG.md:12]: Happens :).

**Comment by barmac** [comment]
I am absolutely delighted with lerna introducing a dependency which breaks a clean install on M3:

```
~/workspace/bpmn-io/dmn-js-2 bump-v16.0.0
❯ npm ci
npm ERR! code 1
npm ERR! path /Users/maciej/workspace/bpmn-io/dmn-js-2/node_modules/nx
npm ERR! command failed
npm ERR! command sh -c node ./bin/post-install
npm ERR! /Users/maciej/workspace/bpmn-io/dmn-js-2/node_modules/nx/src/native/index.js:244
npm ERR!     throw loadError
npm ERR!     ^
npm ERR!
npm ERR! Error: Cannot find module '@nx/nx-darwin-arm64'
npm ERR! Require stack:
npm ERR! - /Users/maciej/workspace/bpmn-io/dmn-js-2/node_modules/nx/src/native/index.js
npm ERR! - /Users/maciej/workspace/bpmn-io/dmn-js-2/node_modules/nx/src/hasher/node-task-hasher-impl.js
npm ERR! - /Users/maciej/workspace/bpmn-io/dmn-js-2/node_modules/nx/src/hasher/task-hasher.js
npm ERR! - /Users/maciej/workspace/bpmn-io/dmn-js-2/node_modules/nx/src/hasher/hash-task.js
npm ERR! - /Users/maciej/workspace/bpmn-io/dmn-js-2/node_modules/nx/src/tasks-runner/run-command.js
npm ERR! - /Users/maciej/workspace/bpmn-io/dmn-js-2/node_modules/nx/src/nx-cloud/utilities/get-cloud-options.js
npm ERR! - /Users/maciej/workspace/bpmn-io/dmn-js-2/node_modules/nx/bin/post-install.js
npm ERR!     at Module._resolveFilename (node:internal/modules/cjs/loader:1144:15)
npm ERR!     at Module._load (node:internal/modules/cjs/loader:985:27)
npm ERR!     at Module.require (node:internal/modules/cjs/loader:1235:19)
npm ERR!     at require (node:internal/modules/helpers:176:18)
npm ERR!     at Object.<anonymous> (/Users/maciej/workspace/bpmn-io/dmn-js-2/node_modules/nx/src/native/index.js:135:29)
npm ERR!     at Module._compile (node:internal/modules/cjs/loader:1376:14)
npm ERR!     at Module._extensions..js (node:internal/modules/cjs/loader:1435:10)
npm ERR!     at Module.load (node:internal/modules/cjs/loader:1207:32)
npm ERR!     at Module._load (node:internal/modules/cjs/loader:1023:12)
npm ERR!     at Module.require (node:internal/modules/cjs/loader:1235:19) {
npm ERR!   code: 'MODULE_NOT_FOUND',
npm ERR!   requireStack: [
npm ERR!     '/Users/maciej/workspace/bpmn-io/dmn-js-2/node_modules/nx/src/native/index.js',
npm ERR!     '/Users/maciej/workspace/bpmn-io/dmn-js-2/node_modules/nx/src/hasher/node-task-hasher-impl.js',
npm ERR!     '/Users/maciej/workspace/bpmn-io/dmn-js-2/node_modules/nx/src/hasher/task-hasher.js',
npm ERR!     '/Users/maciej/workspace/bpmn-io/dmn-js-2/node_modules/nx/src/hasher/hash-task.js',
npm ERR!     '/Users/maciej/workspace/bpmn-io/dmn-js-2/node_modules/nx/src/tasks-runner/run-command.js',
npm ERR!     '/Users/maciej/workspace/bpmn-io/dmn-js-2/node_modules/nx/src/nx-cloud/utilities/get-cloud-options.js',
npm ERR!     '/Users/maciej/workspace/bpmn-io/dmn-js-2/node_modules/nx/bin/post-install.js'
npm ERR!   ]
npm ERR! }
npm ERR!
npm ERR! Node.js v20.11.1

npm ERR! A complete log of this run can be found in: /Users/maciej/.npm/_logs/2024-03-22T14_43_12_767Z-debug-0.log
```

**Comment by barmac** [comment]
I can't run it even after `npm ci --ignore-scripts` :/

**Comment by barmac** [comment]
I updated lerna and it works now.

**Review by barmac — APPROVED** [review]

**Comment by nikku** [comment]
Whoops. Re-added your lerna bump.

**Pull request merged**